tests/test_mock_twilio.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import json
+
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from app.models import db, PhoneNumberVerification
+
+db.create_all()
+
+input = json.loads('''{
+  "caller_name": {
+    "caller_name": "Delicious Cheese Cake",
+    "caller_type": "CONSUMER",
+    "error_code": null
+  },
+  "carrier": null,
+  "country_code": "US",
+  "national_format": "(510) 867-5310",
+  "phone_number": "+15108675310",
+  "add_ons": null,
+  "url": "https://lookups.twilio.com/v1/PhoneNumbers/+15108675310"
+}''')
+  
+output = PhoneNumberVerification.from_twilio_response(input)
+
```

Comment:
to actually save this to the database, i would use `db.session`

```suggestion
db.session.add(output) # this adds your python object to the SQL transaction
db.session.commit() # this executes the transaction, actually running `INSERT` statements

# this queries the phone_number_verifications SQL table, grabbing the first entry
phone_number_entry = db.session.query(PhoneNumberVerification).first() 
```
 I would use `phone_number_entry` in your test instead of `output` Why? `output` is testing that JSON becomes a Python object. We want to test that the Python object can be inserted into the DB (SQL) and that when we ask for it with a query, it's returned from the db with all the correct data.